pkgs/applications/misc/plover/default.nix

```diff
@@ -1,32 +1,32 @@
-{ lib, fetchFromGitHub, python3Packages, wmctrl, qtbase, mkDerivationWith }:
+{ lib, fetchFromGitHub, python3, wmctrl, qtbase, mkDerivationWith }:
 
 {
   stable = throw "plover.stable was removed because it used Python 2. Use plover.dev instead."; # added 2022-06-05
```

Comment:
Is this still the case? 4.0.0 has officially been released and should be considered stable?